test/torchaudio_unittest/transforms/torchscript_consistency_impl.py

```diff
@@ -58,16 +59,6 @@ def test_MuLawDecoding(self):
         tensor = torch.rand((1, 10))
         self._assert_consistency(T.MuLawDecoding(), tensor)
 
-    def test_TimeStretch(self):
```

Comment:
I think we should still keep the old test to validate that we are not introducing bc breaking changes

Reply:
oh nvm just saw that you unified the tests for native complex and pseudo complex inputs!